### DISCUSSION TITLE
Login page popup on navigation

### DISCUSSION THREAD

**Discussion by leonarhv** [drodil/backstage-plugin-qeta#34] [Q&A] [answered]
Hey, 

I'm adopting this plugin in my organization, but I notice that, when I add the plugin, every navigation inside of It forces a Login.
Is this behavior normal? 

Example:

Main Qeta page -> Navigate to New Question page -> Login page Opens

**Comment by drodil** [comment, marked as answer]
Hi. Thanks for the report. I am not sure what could cause this. We are also using this plugin in our organization, and it's also behind authentication, without any problems. What authentication provider are you using? I could try to reproduce this with that. And have you tried using configuration value qeta.allowAnonymous=true? The user is fetched using identityApi and if it cannot get user's name, it will return HTTP 401 unauthorized which may cause redirection to login page.

**Reply by leonarhv** [reply]
Hey @drodil, thanks for the response!

I just tested again the setup and It seems to work. 
I think I actually messed up the IdentityApi setup in the backend, but now It seems to work.

Will mark this as resolved for now.
